tests/llmcompressor/transformers/compression/test_run_compressed.py

```diff
@@ -1,134 +1,182 @@
-import copy
 import shutil
 import tempfile
 import unittest
 
 import torch
-from compressed_tensors import QUANTIZATION_CONFIG_NAME
-from compressed_tensors.compressors import ModelCompressor
-from compressed_tensors.quantization import QuantizationStatus
+from compressed_tensors.linear.compressed_linear import CompressedLinear
+from compressed_tensors.quantization.utils import iter_named_leaf_modules
 from parameterized import parameterized_class
-from transformers import AutoConfig, AutoModelForCausalLM, AutoTokenizer
+from transformers import AutoModelForCausalLM, AutoTokenizer
 from transformers.utils.quantization_config import CompressedTensorsConfig
 
 from tests.testing_utils import parse_params, requires_gpu
 
-CONFIG_DIR = "tests/llmcompressor/transformers/compression/decompression_configs"
+COMPRESSED_LINEAR_CONFIG_DIR = (
+    "tests/llmcompressor/transformers/compression/run_compressed_configs"
+)
 
 
 @requires_gpu
-@parameterized_class(parse_params(CONFIG_DIR))
-class TestDecompression(unittest.TestCase):
+@parameterized_class(parse_params(COMPRESSED_LINEAR_CONFIG_DIR))
+class Test_Decompressed_Linear_Uncompressed_Linear(unittest.TestCase):
```

Comment:
Any reason we're not using CamelCase here?
```suggestion
class TestDecompressedLinearUncompressedLinear(unittest.TestCase):
```